src/murmur/Server.h

```diff
@@ -137,6 +137,7 @@ class Server : public QThread {
 	bool bBonjour;
 	bool bAllowPing;
 	bool allowRecording;
+	uint32_t rollingStatsSeconds;
```

Comment:
Why explicitly a 32-bit integer, i.e. why can't this be a regular `unsigned int`?

Reply:
I was somehow thinking we wouldn't do these types anymore. But I think I misremembered and it was about Qt types.